db.json

```
@@ -0,0 +1,4 @@
+{
+  "latest_id": 0,
+  "memo_lists": []
```

```suggestion
  "memo_lists": {}
```

として、以下のような形式（注：JSONでなくRubyで書いてます）でデータを管理した方がいろいろ効率が良さそうです。

```ruby
memo_lists: {
  1 => {title: 'hello', content: 'hi'},
  2 => {title: 'farewell', content: 'bye'},
}
```